lib/pages/home/contact/contact_page.dart

```diff
@@ -61,59 +62,89 @@ class _MainContactPageState extends State<MainContactPage> {
 
   @override
   Widget build(BuildContext context) {
-    return new Scaffold(
-      appBar: fGetDefaultAppBar(MainContactPage.Title),
-      body: Center(
-          child: new ListView(
-        children: <Widget>[
-          new Card(
-            child: Column(
-              children: <Widget>[
-                fBuildSocialMediaRow(),
-              ],
-            ),
-          ),
-          new Card(
-            child: Column(
-              children: <Widget>[
-                fBuildCompanyName(),
-                fBuildAddressWidget(),
-              ],
-            ),
-          ),
-          new ContactListWidget(
-              mDatabaseKey: gContactsDatabaseKey, mContactsList: gContactsList)
+    return Scaffold(
+      body: CustomScrollView(
+        slivers: <Widget>[
+          fBuildSilverAppBar("assets/images/appbars/main_contacts.png"),
+          SliverFillViewport(
+            delegate: SliverChildBuilderDelegate(
+                (BuildContext context, int index) {
+                  return fBuildBody();
+                },
+                childCount: 1),
+          )
         ],
-      )),
+      ),
+    );
+  }
+
+  Widget fBuildBody() {
+    return Container(
+        padding: EdgeInsets.all(10),
+        child: Column(
+          crossAxisAlignment: CrossAxisAlignment.start,
+          children: <Widget>[
+            fPrintHeadingText(MainContactPage.Title),
+            fBuildSocialMediaRow(),
+            fBuildAddressWidget(),
+            new ContactListWidget(
+                mDatabaseKey: gContactsDatabaseKey,
+                mContactsList: gContactsList)
+          ],
+        ));
+  }
+
+/*
+  @override
+  Widget build(BuildContext context) {
+    return new Scaffold(
+      appBar: fBuildAppBar("assets/images/about_us_top_image.png"),
+      body: SingleChildScrollView(
+          child: new Container(
+              padding: EdgeInsets.all(10),
+              child: Column(
+                crossAxisAlignment: CrossAxisAlignment.start,
+                children: <Widget>[
+                  fPrintHeadingText(MainContactPage.Title),
+                  fBuildSocialMediaRow(),
+                  fBuildAddressWidget(),
+                  new ContactListWidget(
+                      mDatabaseKey: gContactsDatabaseKey,
+                      mContactsList: gContactsList)
+                ],
+              ))),
     );
   }
+*/
 
   Widget fBuildSocialMediaRow() {
+    double iconSize = 70.0;
+
     return new Container(
-        padding: EdgeInsets.all(10.0),
+        padding: EdgeInsets.only(top: 10.0, bottom: 10.0),
         child: Row(
           mainAxisAlignment: MainAxisAlignment.spaceEvenly,
           children: <Widget>[
             IconButton(
```

Comment:
Można by osobny Widget expandujący IconButton zrobić do którego by się przekazywało ściężkę do ikonki i url'a

Reply:
BussinessIconButton extends IconButton

Reply:
Albo tak niżej
Do dodawania ikony funkcję zwracającą Widget dodać w print.dart

Reply:
i lajk it